commit f8c5811911e413640472022be8e5c77bbbaa995d (refs/changes/00/5100/1)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-11-22T11:01:46+00:00 (11 months ago)
    
    MANTA-4021 wrasse's nodeunit dep cannot install with old node (fix build)

diff --git a/Makefile b/Makefile
index 74484bd..0992d47 100644
--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,7 @@ NODEUNIT	:= ./node_modules/.bin/nodeunit
 #
 # Files
 #
-JS_FILES	:= $(shell ls *.js) $(shell find lib test -name '*.js')
+JS_FILES	:= $(shell ls *.js) $(shell find lib -name '*.js')
 JSL_CONF_NODE	 = tools/jsl.node.conf
 JSL_FILES_NODE   = $(JS_FILES)
 JSSTYLE_FILES	 = $(JS_FILES)
@@ -90,7 +90,6 @@ release: all docs $(SMF_MANIFESTS)
 		$(ROOT)/package.json \
 		$(ROOT)/sapi_manifests \
 		$(ROOT)/smf \
-		$(ROOT)/test \
 		$(RELSTAGEDIR)/root/opt/smartdc/$(NAME)
 	mv $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/build/scripts \
 	    $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/boot
diff --git a/package.json b/package.json
index da33c13..36027e4 100644
--- a/package.json
+++ b/package.json
@@ -23,8 +23,5 @@
         "rimraf": "2.2.6",
         "vasync": "1.4.3"
     },
-    "scripts": {
-        "test": "nodeunit test/*.test.js"
-    },
     "license": "MPL-2.0"
 }
